chore: Run "c-code-style" astyle formatter on source/header files

Dazza0 opened the pull request:
Ran `astyle --options="astyle-code-format.cfg" "./*.c,*.h"`. Caught some trailing whitespaces.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/MaJerle/lwshell?pullRequest=19) <br/>All committers have signed the CLA.

Dazza0 committed:
chore: Run "c-code-style" astyle formatter on source/header files

Ran `astyle --options="astyle-code-format.cfg" "./*.c,*.h"`. Fixed some
trailing whitepsaces and indentation.
